boards/beagle/beaglebone_ai64/Kconfig.beaglebone_ai64

```diff
@@ -0,0 +1,8 @@
+# Copyright (C) 2023 BeagleBoard.org Foundation
+# Copyright (C) 2023 S Prashanth
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_BEAGLEBONE_AI64_J721E_R5
+	bool "BeagleBone AI-64 R5"
+	select SOC_J721E_R5
```

Comment:
select x if *use board name with socs*

Reply:
Will fix.

Also, I copied this from `boards/ti/sk_am62/Kconfig.sk_am62`, I guess that will need fixed the same way, I'll go do that too.